app/src/interfaces/useObservationApi.interface.ts

```diff
@@ -37,3 +37,34 @@ export interface IGetObservationResponse {
   id: number;
   data: ICreateObservationRequest;
 }
+
+export interface IGetBlocksListResponse {
+  id: number;
+  block_id: number;
+  number_of_observations: number;
+  start_time: string;
+  end_time: string;
+}
+
+export interface IGetObservationsListResponse {
+  blocks: IGetBlocksListResponse[];
+}
+
+/**
+ * Create blockObservation post object.
+ *
+ * @export
+ * @interface ICreateBlockObservationPostRequest
+ */
+export interface ICreateBlockObservationPostRequest {
```

Comment:
this does not seem correct based on the new data you are sending to the endpoint, I think it needs to be updated to match

Reply:
I'm going to leave this for now .... until we know more about the requirements of the other observation types.

Reply:
no what I meant is that this doesnt match what is being sent and needs to be updated to match this structure:
```
observation_type,
observation_post_data: {
  block_name
  start_datetime
  end_datetime
  observation_count
  observation_data: {
    metaData
    tableData: {
      data
    }
  }
}
```

Reply:
oh i see!